makerdao/governance-portal-v2
# Adding polling comments & Removing votes from ballot

### Link to Shortcut ticket:
https://app.shortcut.com/dux-makerdao/story/44/adding-comments-to-polling
https://app.shortcut.com/dux-makerdao/story/810/allow-removing-poll-votes-from-ballot

### What does this PR do?
Adds polling comments. 
- Creates two buttons to sign the message + post the transaction
- Creates API endpoints for polling comments
- Refactors comments into it's own module
- Display polling comments + cross data with vote history 
- Adds sharing button on the comment list (disabled for now)
- Allow to remove votes from ballot
- Fix a bug where the total deposited on the chief was not updated
- Allows to post comments on executives on other networks different than mainnet


### Steps for testing:
Go to a poll:
- put some comment and submit your ballot
- check all the executive comments

### Screenshots (if relevant):
![image](https://user-images.githubusercontent.com/1152768/145596885-e23ebbf8-74b7-4160-83f2-326cc8ac9835.png)

### Any additional helpful information?:
To connect to the DB locally you will need the environment variables for mongoDb, kindly ask for them

### Add a GIF:
<img src="https://media3.giphy.com/media/Ve1KReZyJLTakI9GRI/giphy.gif"/>
![image](https://user-images.githubusercontent.com/1152768/145597696-65365a85-42d0-45ae-89ee-fc3228a0083d.png)
